packages/runner/src/context.ts

```diff
@@ -104,10 +104,18 @@ export function createTestContext(
 
   context.task = test
 
-  context.skip = (note?: string) => {
+  context.skip = (condition?: boolean | string, note?: string): never => {
+    if (typeof condition === 'boolean' && !condition) {
```

Comment:
Isn't this equivalent to:

```suggestion
    if (condition === false) {
```